src/main/java/frc/robot/Constants.java

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.SwerveDriveKinematics;
+import edu.wpi.first.math.util.Units;
+
+/**
+ * The Constants class provides a convenient place for teams to hold robot-wide numerical or boolean
+ * constants. This class should not be used for any other purpose. All constants should be declared
+ * globally (i.e. public static). Do not put anything functional in this class.
+ *
+ * <p>It is advised to statically import this class (or one of its inner classes) wherever the
+ * constants are needed, to reduce verbosity.
+ */
+public final class Constants {
+
+    public static final class MotorDefaults{
+        //Constants to use as default values for Motor Controllers
+        public static final int kCurrentLimit = 40;
+        public static final double kOpenLoopRampRate = 0.2;
+    }
+    
+    public static final class DriveConstants{ //TODO: Update these constants with physical values
+        public static final double kPhysicalMaxSpeed = 1; //Max drivebase speed in meters per second
+        public static final double kPhysicalMaxAngularSpeed = 1; //Max drivebase angular speed in radians per second
+
+        public static final double kTrackWidth = Units.inchesToMeters(18.75); //Distance between right and left wheels
+        public static final double kWheelBase = Units.inchesToMeters(18.75); //Distance between front and back wheels
+        public static final SwerveDriveKinematics kDriveKinematics = new SwerveDriveKinematics( //Creates robot geometry using the locations of the 4 wheels
+            new Translation2d(kWheelBase / 2, -kTrackWidth / 2), 
+            new Translation2d(kWheelBase / 2, kTrackWidth /2),
+            new Translation2d(-kWheelBase / 2, -kTrackWidth / 2),
+            new Translation2d(-kWheelBase / 2, kTrackWidth / 2));
+    }
+
+    public static final class SwerveModuleConstants{ //TODO: Update these constants with physical values
+        public static final double kPhysicalMaxSpeed = 1; //Max module speed in meters per second
+        public static final double kWheelDiameterMeters = Units.inchesToMeters(4);
+        public static final double kDriveMotorGearRatio = 6.75 / 1;
```

Comment:
SDS's code uses a value of (14.0 / 50.0) * (27.0 / 17.0) * (15.0 / 45.0) for drive gear reduction

https://github.com/SwerveDriveSpecialties/swerve-lib/blob/55f3f1ad9e6bd81e56779d022a40917aacf8d3b3/src/main/java/com/swervedrivespecialties/swervelib/SdsModuleConfigurations.java#L55